### ISSUE TITLE
Chorus in FOTR

### ISSUE THREAD

**Issue by genekellyjr** [genekellyjr/LOTR-Subs#19]
Monoverantus (https://www.youtube.com/@monoverantus/videos) has a great series on the music and actually has figured out what the chorus is saying in time.

The catch is it often doesn't match the text of the songs elsewhere. Like https://youtu.be/q4xh7aFxcPs?si=a17XY1Ssjt7Y2IRE&t=139 and https://lotr.fandom.com/wiki/The_Revelation_of_the_Ringwraiths. I assume the songs have been sliced and diced for the movie, and he is obviously knowledgeable so I'm trusting his translations.

Basically I see in the linked song/actual lyrics that the text is diced hard - but the arrangement is supposed to represent coherent ideas so I'm happy to give these a lot of leeway even if they're missing words. But I would put missing words in brackets or something, if someone can actually suss out what is sung vs what is meant.
